CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Dunno what a good cutting-off point would be w/r/t lowest supported CMake version, just picked the lowest 3.x version.

Reply:
- XP CI ships CMake 3.12.2.
- Ubuntu 18.04 ships 3.10.2.
- Can't check what Ubuntu 16.04 ships, Ubuntu removed xenial from their package search.